lib/validate.js

```diff
@@ -39,6 +59,22 @@ module.exports = function validateMiddleware (opts) {
     }
   });
 
+  assert(
+    isObject(opts.joiValidateOptions),
```

Comment:
These should probably be optional. If so, they need to wrapped in an _if_. Thoughts?